CAM/RenderScriptCams.md

```diff
@@ -8,22 +8,15 @@ ns: CAM
 void RENDER_SCRIPT_CAMS(BOOL render, BOOL ease, int easeTime, BOOL p3, BOOL p4);
```

Comment:
Where's `easeCoordsAnim`, which is `p3`? This is why the checks are failing.

Reply:
Oh, now I see thanks